nav2_rviz_plugins/include/nav2_rviz_plugins/route_tool.hpp

```diff
@@ -0,0 +1,126 @@
+// Copyright (c) 2024 Leidos
```

Comment:
this is peak nitpick but (c) should be uppercase (C) to represent the Copyright symbol. I think this needs to be updated in most of the files

Reply:
It looks like the rest of nav2 uses a lowercase c, so i left this for the sake of consistency

Reply:
100% should be a capital, time to create an official Nav2 issue for improper use of Copyright as a capitalized label (your choice but this is a standard `It is designed as an encircled capital C © to refer to the “copyright.”` )